backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -623,6 +643,7 @@ public static Bundle createFhirBundle(
     var bundle =
         new Bundle()
             .setType(BundleType.MESSAGE)
+            .setTimestamp(currentDate)
```

Comment:
should we just generate the timestamp on bundle creation instead of passing it in?

Reply:
Without passing in the current date, it makes it difficult to test.